README.md

```diff
@@ -5,6 +5,7 @@ Core functionalities of the Open Systems Pharmacology Suite.
 ## Code Status
 [![NuGet version](https://img.shields.io/nuget/v/OSPSuite.Core.svg?style=flat)](https://www.nuget.org/packages/OSPSuite.Core)
 [![Build status](https://ci.appveyor.com/api/projects/status/w3ecxmp3grryc5ts/branch/master?svg=true&passingText=master%20-%20passing)](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-core/branch/master)
+[![Coverage Status](https://coveralls.io/repos/github/rwmcintosh/OSPSuite.Core/badge.svg?branch=master)](https://coveralls.io/github/rwmcintosh/OSPSuite.Core?branch=master)
```

Comment:
That line will change. First I need you to configure a coveralls project for the OSPSuite.Core repository.